### DISCUSSION TITLE
Resolution of a presentation or shared desktop page shown in the room is hard to read

### DISCUSSION THREAD

**Discussion by albertjan78** [mozilla/hubs#3899] [General Questions & Discussion] [answered]
Dear All,

When showing PDF presentations or sharing the desktop in a room the resolution is poor. when looking from a larger distance on desktop, Oculus Quest or HTC VIVE it's hard to read the text from a larger distance. Anyone an idea what could be the cause?

**Comment by CrimsonZA** [comment]
Would you say it has a wavy effect or that the texturing has extremely sharp filtering ?

**Reply by albertjan78** [reply]
Hi CrimsonZA,

Thanks for your reply and apologies for my late response. You could say wavy effect. it feels a bit like an antialising effect around the letters in the presentation/shared desktop. the letters become sharp only from a short distance.

**Reply by johnshaughnessy** [reply]
This sounds like a sampling problem on VR headsets. Unfortunately we have no tech in place to optimize text rendering for these headsets (especially from a screenshare). I'm going to mark this question as resolved because we don't have a solution to make the text clear from a distance at this time.

**Comment by johnshaughnessy** [comment, marked as answer]
Answered in https://github.com/mozilla/hubs/discussions/3899#discussioncomment-554725
